KarhooUISDK/Screens/CheckoutScreen/Components/Loyalty/LoyaltyView.swift

```diff
@@ -41,12 +39,33 @@ struct LoyaltyView: View {
     }
     
     @ViewBuilder
-    private var earnContent: some View {
-        let pointsEarnedText = String(
+    private var viewOverOrSeparator: some View {
```

Comment:
why not call it earnView? viewOverOrSeparator doesn't sound very clear as to what that view's purpose is

Reply:
Because in case of EarnDisalbed = true this view will contain only title "Reward Points". So i went for name viewOverOrSeparator because it is correct no matter what we will put inside this view.